src/wasm-lib/kcl/src/std/loft.rs

```diff
@@ -142,19 +145,34 @@ async fn inner_loft(
         }));
     }
 
-    let id = exec_state.next_uuid();
-    args.batch_modeling_cmd(
-        id,
-        ModelingCmd::from(mcmd::Loft {
-            section_ids: sketches.iter().map(|group| group.id).collect(),
-            base_curve_index,
-            bez_approximate_rational,
-            tolerance: LengthUnit(tolerance.unwrap_or(default_tolerance(&args.ctx.settings.units))),
-            v_degree,
-        }),
-    )
-    .await?;
+    let id: uuid::Uuid = exec_state.next_uuid();
+    let resp = args
+        .send_modeling_cmd(
+            id,
+            ModelingCmd::from(mcmd::Loft {
+                section_ids: sketches.iter().map(|group| group.id).collect(),
+                base_curve_index,
+                bez_approximate_rational,
+                tolerance: LengthUnit(tolerance.unwrap_or(default_tolerance(&args.ctx.settings.units))),
+                v_degree,
+            }),
+        )
+        .await?;
```

Comment:
Discussion started with the engine team to see if we could get an explicit solid_id field as part of the Loft command, this way it we could keep batching this one. See https://github.com/KittyCAD/modeling-app/commit/c9f7336edc9e7f6c26cf8c02b30fccf61e26f966

Reply:
This is the PR we need merged https://github.com/KittyCAD/engine/pull/2969